spanner/jdbc/src/main/java/com/example/spanner/jdbc/LoadCsvExample.java

```diff
@@ -0,0 +1,290 @@
+  public static boolean validNonHeaderField(CSVRecord record, int index) {
+    return !hasHeader && record.get(index) != null;
+  }
+
+  /** Write CSV file data to Spanner using JDBC Mutation API **/
+  public static void writeToSpanner(Iterable<CSVRecord> records, String tableName, CommandLine cmd)
+      throws SQLException {
+    System.out.println("Writing data into table...");
+    List<Mutation> mutations = new ArrayList<>();
+    for (CSVRecord record : records) {
+      int index = 0;
+      WriteBuilder builder = Mutation.newInsertOrUpdateBuilder(tableName);
+      for (String columnName : tableColumns.keySet()) {
+        // Iterates through columns in order. Assumes in order columns when no headers provided.
+        SpannerDataType columnType = tableColumns.get(columnName);
+        String recordValue = null;
+        if (validHeaderField(record, columnName)) {
+          recordValue = record.get(columnName).trim();
+        } else if (validNonHeaderField(record, index)) {
+          recordValue = record.get(index).trim();
+          index++;
+        }
+        if (recordValue != null) {
+          switch (columnType) {
+            case STRING:
+              builder.set(columnName).to(recordValue);
+              break;
+            case BYTES:
+              builder.set(columnName).to(Byte.parseByte(recordValue));
+              break;
+            case INT64:
+              builder.set(columnName).to(Integer.parseInt(recordValue));
+              break;
+            case FLOAT64:
+              builder.set(columnName).to(Float.parseFloat(recordValue));
+              break;
+            case BOOL:
+              builder.set(columnName).to(Boolean.parseBoolean(recordValue));
+              break;
+            case DOUBLE:
+              builder.set(columnName).to(Double.parseDouble(recordValue));
+              break;
+            case NUMERIC:
+              builder.set(columnName).to(Value.numeric(BigDecimal.valueOf(
+                  Double.parseDouble(recordValue))));
+              break;
+            case LONG:
+              builder.set(columnName).to(Long.parseLong(recordValue));
+              break;
+            case DATE:
+              builder.set(columnName).to(com.google.cloud.Date.parseDate(recordValue));
+              break;
+            case TIMESTAMP:
+              builder.set(columnName).to(com.google.cloud.Timestamp.parseTimestamp(recordValue));
+              break;
+            default:
+              System.out.print("Invalid Type. This type is not supported.");
+          }
+        }
+      }
+      mutations.add(builder.build());
+    }
+    CloudSpannerJdbcConnection spannerConnection = connection
+        .unwrap(CloudSpannerJdbcConnection.class);
+    spannerConnection.write(mutations);
+    spannerConnection.close();
+    System.out.println("Data successfully written into table.");
+  }
+
+  static void loadCsv(String projectId, String instanceId, String databaseId, String tableName,
```

Comment:
How will a user run this sample? Please also see [Method Structure](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/SAMPLE_FORMAT.md#method-structure)

Reply:
I was under the impression that the example of how users would run the sample would be just to follow the same setup as provided in the test I wrote for this function, but let me know if you mean something else.

Also I'm not exactly sure what kind of main function the Method Structure is referring to as I haven't seen any of the other example files have main functions?

Reply:
@averikitsch  I don't think Method Structure applies here, as this is an example similar to other existing examples in the same directory.

Reply:
We usually provided an overloaded method, "Any declared function arguments should include a no-arg, main method with examples for how the user can initialize the method arguments and call the entrypoint for the snippet. ". Also does the [README](https://github.com/GoogleCloudPlatform/java-docs-samples/tree/ab0e5d10706468c27dabf27c7b2b20e2c684d8f4/spanner/jdbc) properly describe how to run the snippet?

Reply:
Is there an example that you could point me to where this overloaded method is implemented for the other example files? 

Also will update the README 👍🏼

Reply:
[Example 1 using an overloaded method](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/secretmanager/src/main/java/secretmanager/CreateSecret.java#L28) and [Example 2 using a main method](https://github.com/googleapis/java-bigquery/blob/master/samples/snippets/src/main/java/com/example/bigquery/CopyTable.java#L30)